optimas/generators/ax/developer/multitask.py

```diff
@@ -274,11 +282,14 @@ def suggest(self, num_points: Optional[int]) -> List[dict]:
                     var.name: arm.parameters.get(var.name)
                     for var in self._varying_parameters
                 }
-                # SH for VOCS standard these will need to be 'variables'
-                # For now much match the trial parameter names.
+                # SH We can use a discrete var here in vocs (converted for now to trial parameters)
+                # But unlike varying parameters the name refers to a fixed generator concept.
+                for trial_param in self._custom_trial_parameters:
+                    if trial_param.name == "trial_type":
```

Comment:
I am a bit confused ; does this if condition ever become true?
Given that we have:
```
        custom_trial_parameters = [
            TrialParameter("arm_name", "ax_arm_name", dtype="U32"),
            TrialParameter("ax_trial_id", "ax_trial_index", dtype=int),
        ]
```

Reply:
Actually: looking at the rest of the code in this PR, it seems that this would be a case where we use `set_fidelity_param` on the generator, no?